apps/frontend/src/pages/access-management/libs/helpers/format-user-date/format-user-data.helper.ts

```diff
@@ -0,0 +1,14 @@
+const formatUserDate = (createdAt: string): string => {
```

Comment:
maybe it would be better to generalize formatUserDate into a formatDate function. this way, it can be used in other contexts as well